README.md

```diff
@@ -106,6 +106,11 @@ Results on ``title`` or ``content`` are weighted in the results. Search result s
 
 Enable wildcard after words. So when searching for for example  ``car`` it will also match ``carbon``. 
 
+### exclude_feature_enabled
+This feature excludes some rows from being  returned. Enable this when you have a flag in your model which determines whether this record must be returned in search queries or not. By default this feature is disabled.
```

Comment:
Double space :)

Reply:
Also a small example would be nice i think since.

Reply:
OK I'll add and example too